authorization/policy.csv

```diff
@@ -0,0 +1,4 @@
+p, anonymous, 59fd884d8f6b180001f5b4e2:/auth/login, POST
```

Comment:
What is it for?

Reply:
It is for allowing anonymous users to login into the app with id 59fd884d8f6b180001f5b4e2.